packages/kit/src/runtime/server/page/render.js

```diff
@@ -316,10 +291,31 @@ export async function render_response({
 			opts.push(`hydrate: {\n\t\t\t\t\t${hydrate.join(',\n\t\t\t\t\t')}\n\t\t\t\t}`);
 		}
 
-		// prettier-ignore
+		const streaming = needs_streaming
+			? `window.$__sveltekit__ = window.$__sveltekit__ || {};
```

Comment:
noted in https://github.com/sveltejs/kit/pull/9018#discussion_r1108175556 but we should change one or the other